MSProject/LibraryCollectionView/CollectionCells/WBBookCollectionViewCell.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  WBBookCollectionViewCell.swift
+//  MSProject
+//
+//  Created by Matias Spinelli on 10/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class WBBookCollectionViewCell: UICollectionViewCell, NibLoadable {
+
+    @IBOutlet weak var bookImage: UIImageView!
+    @IBOutlet weak var bookTitle: UILabel!
+    @IBOutlet weak var bookAuthor: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        layer.cornerRadius = 5
+        backgroundColor = .white
+    }
+    
+    var bookCellViewModel: WBBookCellViewModel? {
```

Comment:
I don't think that a View should have a reference to a ViewModel, as it would be a bypass of the MVVM structure. That should be the responsibility of a ViewController. As your active ViewController is the one for the LibraryCollection, it's kinda telling you that in any case, you should have the reference there or in the LibraryCollection ViewModel.

Regardless, I think the best implementation here would be to set the cell information in the ViewController, and to fetch all the needed data in its ViewModel. It's too much logic for a View to handle in MVVM.

💪